assignments/arrays.js

```diff
@@ -63,34 +63,118 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+//console.log(inventory[32]);
+//console.log(Object.values(inventory[32]));
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
+//console.log(inventory[32].car_year );
+//console.log(inventory[32].car_make );
+//console.log(inventory[32].car_model );
+//console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+
+
 let lastCar = 0;
-console.log();
+//let lastCar = inventory[inventory.length-1];
+//console.log(inventory[inventory.length-1].car_make );
+//console.log(inventory[inventory.length-1].car_model );
+let finCar = `Last car is a ${inventory[inventory.length-1].car_make} ${inventory[32].car_model}`
+
+
+console.log(finCar);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+
+
+
+let carModels = inventory.sort(compare);
+console.log(carModels);
+
+
+function compare(a,b) {
+  if (a.car_model < b.car_model)
+    return -1;
+  if (a.car_model > b.car_model)
+    return 1;
+  return 0;
+}
+
+inventory.sort(function(a,b) {
+    var modelA = a.car_model.toUpperCase();
+    var modelB = b.car_model.toUpperCase();
+    if(modelA<modelB) {
+        return -1;
+    }
+    if (modelA>modelB) {
+        return 1;
+    }
+    return 0;
+});
+
+
+
+
+
+
 
 // ==== Challenge 4 ====
```

Comment:
I had to comment out line 140 because it was returning two answers. But overall you got the result that was asked for.